index.html

```diff
@@ -1111,11 +1111,18 @@ <h3>Proofs (Signatures)</h3>
         <dl>
           <dt><var>proof</var></dt>
           <dd>
-The method used for a mathematical proof varies by representation language and
-the technology used.  For example, if digital signatures are used for the
-proof mechanism, the <code>proof</code> <a>property</a> is expected to have a
-value that is a set of name-value pairs including at least a signature, a
-reference to the signing entity, and a representation of the signing date.</dd>
+No single proof mechanism is currently standardised or recommended for verifiable
+credentials. The method used for a proof SHOULD be stated in the <code>type</code>
```

Comment:
If not MUST then how does a Verifier know what the Proof will contain?
We have been around this many times already, and made the types mandatory in all other properties such as ToU, so that the verifier can determine on first parsing whether it supports the type or not.